packages/Makefile

```diff
@@ -1,7 +1,8 @@
 
-YB_VERSION ?= 2.1.1.0
+YB_VERSION ?= 2.1.5.0
+YB_CLIENT_VERSION ?= 2.1
 SERVER_TAR_NAME = yugabyte-$(YB_VERSION)-linux.tar.gz
-CLIENT_TAR_NAME = yugabyte-client-$(YB_VERSION)-linux.tar.gz
+CLIENT_TAR_NAME = yugabyte-client-$(YB_CLIENT_VERSION)-linux.tar.gz
 DOWNLOADS_URL = "https://downloads.yugabyte.com"
 
 YB_SERVER_RPM_REVISION = 2
```

Comment:
minor: maybe add some comments to the PR about how the rpm revision value corresponds to the client version. For ex: if we change client version to 2.2, what would we change the rpm_revision to? What is the command line for users to install a specific rpm version?

Reply:
Yes, I will add those details in the README.md itself.